### DISCUSSION TITLE
Container management utility not found

### DISCUSSION THREAD

**Discussion by antaradey-itk** [eclipse-leda/leda-distro#97] [Q&A] [answered]
I am trying to add the private registry for the cloudconnector, but **sdv-kanto-ctl** command not found.
Can anyone give any insight on this of what might be missing?

![image](https://user-images.githubusercontent.com/129940848/235698098-7c34f13a-3079-43eb-ba11-4a3474e31ce6.png)

**Comment by mikehaller** [comment, marked as answer]
The script has been recently added and is only available in the latest builds. Could it be that you're using an outdated version of the Leda image?

Check sdv-health for version number. Should be 0.1.0-M1 or higher

Another cause may be that the script is only available on the full image (which is booted by default), but your current booted partition is using the sdv-image-minimal (second boot partition)

Could you check with mount whether root partition is 4 (full) or 5 (minimal)?

It should be possible to download the script from GitHub and copy it to the device - as a workaround in case you don't want to reinstall a newer base image.

**Reply by antaradey-itk** [reply]
I was using an outdated version (pre-release 0.0.5). 
Version should be 0.1.0-M1+
